src/states_screens/track_info_screen.hpp

```diff
@@ -46,6 +46,10 @@ class TrackInfoScreen : public GUIEngine::Screen,
 
     bool m_record_this_race;
 
+    bool m_is_soccer;
```

Comment:
These new variables should be initialized with a default value in the constructor, yet unless I missed something that does not seem to be the case